extras/github/test_docker.py

```diff
@@ -80,7 +83,7 @@ def test_release_candidate_non_default_python(self):
             'GITHUB_EVENT_DEFAULT_BRANCH': 'master',
             'GITHUB_EVENT_NUMBER': '',
             'MATRIX_PYTHON_IMPL': 'python',
-            'MATRIX_PYTHON_VERSION': '3.11',
+            'MATRIX_PYTHON_VERSION': '3.10',
```

Comment:
Maybe create a `NON_DEFAULT_PYTHON_VERSION`?

Reply:
Makes sense, I'll do that.

Reply:
Thanks!